### PR TITLE
Zib-DOSScore

### PR DESCRIPTION
**Pull request by Ashwin-nictiz**
TODO: zib compliance tool

### CODE REVIEW

File: resources/zib/zib-DOSScore.xml [hunk 1]
```diff
@@ -0,0 +1,545 @@
...
+    <element id="Observation.component:dozesOff">
+      <path value="Observation.component" />
+      <sliceName value="dozesOff" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:dozesOff.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
```
Comment (LuudSlagter) [inline resources/zib/zib-DOSScore.xml:137]: These codes are not from SNOMED, but from the (internal) code system ScoreObservaties.

File: resources/zib/zib-DOSScore.xml [hunk 2]
```diff
@@ -0,0 +1,545 @@
...
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:dozesOff.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="DozesOff" />
+      <definition value="DOS observation: patient dozes off during conversation or activities.&#xD;&#xA;&#xD;&#xA;Score:&#xD;&#xA;0: never&#xD;&#xA;1: sometimes-always&#xD;&#xA;-: don't know" />
+      <alias value="ZaktWeg" />
+      <type>
+        <code value="integer" />
```
Comment (LuudSlagter) [inline resources/zib/zib-DOSScore.xml:148]: Each component should have a minimum and maximum allowed value of 0 and 1 respectively.

**Review by LuudSlagter — CHANGES_REQUESTED** [review]

File: resources/zib/zib-DOSScore.xml (hunk 1 above)
Comment (Ashwin-nictiz) [reply resources/zib/zib-DOSScore.xml]: Fixed

File: resources/zib/zib-DOSScore.xml (hunk 2 above)
Comment (Ashwin-nictiz) [reply resources/zib/zib-DOSScore.xml:148]: Done

File: resources/nl-core/nl-core-DOSScore.xml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  <purpose value="A derived profile from [zib-DOSScore](http://nictiz.nl/fhir/StructureDefinition/zib-DOSScore) to provide a version better suited for implementation purposes. This profile augments the base profile with elements found in the various use cases that have adopted the zib." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="Observation" />
+  <baseDefinition value="http://nictiz.nl/fhir/StructureDefinition/zib-DOSScore" />
+  <derivation value="constraint" />
+  <differential>
```
[the remark this answers is not in the document]
Comment (Ashwin-nictiz) [reply resources/nl-core/nl-core-DOSScore.xml:26]: Fix

File: examples/nl-core-DOSScore-01.xml [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+   </subject>
+   <effectiveDateTime value="2016-12-19T10:35:00+01:00"/>
+   <valueInteger value="4"/>
+   <note>
+      <text value="Patiënt is mogelijk delirant."/>
+   </note>
+   <component>
+      <code>
+         <coding>
+            <system value="urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.0.1"/>
```
Comment (Ashwin-nictiz) [inline examples/nl-core-DOSScore-01.xml:27]: There is a error about the system value:

  -  error at Observation.component[0].code.coding[0] (26, 18):
     Unknown Code urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.0.1#18007006 in urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.0.1 for 'urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.0.1#18007006'

This need to be fixed in the Validation tool

**Comment by LuudSlagter** [comment]
Looking good (with the exception of the remaining validation errors)!

**Review by LuudSlagter — APPROVED** [review]

**Pull request merged**